lists/sa.csv

```diff
@@ -398,3 +398,9 @@ http://www.yementodaytv.net/,NEWS,News Media,2016-04-19,citizenlab,
 http://www.youtube.com/verify_controversy?next_url=/watch%3fv%3dqmodvun16q4,NEWS,News Media,2014-04-15,citizenlab,
 http://www.zawadj.net,DATE,Online Dating,2014-04-15,citizenlab,
 http://yawmyat.blogspot.com,NEWS,News Media,2014-04-15,citizenlab,
+https://www.aljazeera.com/,NEWS,News Media,2018-06-14,frekeeh,

```

Comment:
I would suggest moving this to the global list. We currently have aljazeera.net, but that doesn't seem to be the canonical domain anymore.

@agrabeli thoughts?

Reply:
@hellais We should probably be testing both aljazeera.com and aljazeera.net via the global test list (sorry for seeing this question so late!).